### ISSUE TITLE
Tidy up contributor guide / local development environment

### ISSUE THREAD

**Issue by mattwynne** [cucumber/cucumber-expressions#17]
We currently have the Makefiles imported from the monorepo, which can build all language versions at once. Do we still need those?

Should we provide a Docker environment (or multiple per-language environments) to help contributors build the projects?

The `CONTRIBUTING.md` file is empty.

**Comment by aurelien-reeves** [comment]
We may first fill the `CONTRIBUTING.md` file, maybe taking some inspiration from the one from `cucumber-ruby`: https://github.com/cucumber/cucumber-ruby/blob/main/CONTRIBUTING.md ?

**Comment by mattwynne** [comment]
Yes, I've also had good feedback on the intro to [the one in cucumber-js](https://github.com/cucumber/cucumber-js/blob/main/CONTRIBUTING.md) when I've done some 1:1 onboarding sessions.

Maybe we can put a contributing doc into the [template repo](https://github.com/cucumber/.github)?

**Comment by aurelien-reeves** [comment]
> 
> 
> Yes, I've also had good feedback on the intro to [the one in cucumber-js](https://github.com/cucumber/cucumber-js/blob/main/CONTRIBUTING.md) when I've done some 1:1 onboarding sessions.
> 
> Maybe we can put a contributing doc into the [template repo](https://github.com/cucumber/.github)?

Yes, definitely
